src/strategies/a51-vault-balance/index.ts

```diff
@@ -0,0 +1,106 @@
+import { getAddress } from '@ethersproject/address';
+import { Multicaller, subgraphRequest } from '../../utils';
+
+export const author = 'rafaqat11';
+export const version = '0.1.0';
+
+const A51_VAULT_SUBGRAPH_URL = {
+  '137':
+    'https://api.thegraph.com/subgraphs/name/hamzabhatti125/unipilot-quickswap-stats'
+};
+
+const vaultABI = {
+  inputs: [],
+  name: 'getPositionDetails',
+  outputs: [
+    { internalType: 'uint256', name: 'amount0', type: 'uint256' },
+    { internalType: 'uint256', name: 'amount1', type: 'uint256' },
+    { internalType: 'uint256', name: 'fees0', type: 'uint256' },
+    { internalType: 'uint256', name: 'fees1', type: 'uint256' },
+    { internalType: 'uint128', name: 'baseLiquidity', type: 'uint128' },
+    { internalType: 'uint128', name: 'rangeLiquidity', type: 'uint128' }
+  ],
+  stateMutability: 'nonpayable',
+  type: 'function'
+};
```

Comment:
Hi @RAFAQAT11 Could you create separate PRs for each strategy? it will help us merge faster :)

Reply:
@ChaituVR, I have created separate PRs.